ui/src/app/machines/views/MachineDetails/MachineStorage/CacheSetsTable/CacheSetsTable.test.tsx

```diff
@@ -38,11 +40,61 @@ describe("CacheSetsTable", () => {
     const store = mockStore(state);
     const wrapper = mount(
       <Provider store={store}>
-        <CacheSetsTable systemId="abc123" />
+        <CacheSetsTable canEditStorage systemId="abc123" />
       </Provider>
     );
 
     expect(wrapper.find("tbody TableRow").length).toBe(1);
     expect(wrapper.find("TableCell").at(0).text()).toBe(cacheSet.name);
   });
+
+  it("can delete a cache set", () => {
+    const disk = diskFactory({
+      type: "cache-set",
```

Comment:
Hah, same question again, this looks like an enum (and can see there are a few types). Not necessary to update this branch, but maybe add a card?

Reply:
yep an enum would be a great idea, adding a card now